### ISSUE TITLE
Possible bug in _create_remote_acquisition

### ISSUE THREAD

**Issue by carlkesselman** [micro-manager/pycro-manager#402]
### Bug report

**Bug summary**

It looks like core is not being properly initialized in _create_remote_acquition when an alternative port is used, for example via start_headless
**Code for reproduction**
The call to Core() in this code should probably include the port argument.

```python
    def _create_remote_acquisition(self, **kwargs):
        core = Core()
        acq_factory = JavaObject(
            "org.micromanager.remote.RemoteAcquisitionFactory", args=[core]
        )
        show_viewer = kwargs['show_display'] == True and (kwargs['directory'] is not None and kwargs['name'] is not None)

        self._remote_acq = acq_factory.create_acquisition(
            kwargs['directory'],
            kwargs['name'],
            show_viewer,
            kwargs['saving_queue_size'],
            kwargs['core_log_debug'],
        )
```

**Expected outcome**

**Actual outcome**

**Version Info**

  * Operating system: OS X
  * pycromanager version: 17.5 (latest)
  * MicroManager version: nightly build
  * Python version: 3.10
  * Python environment (command line, IDE, Jupyter notebook, etc) pycharm

**Comment by carlkesselman** [comment]
working on a fix... however, it looks a bit more complicated then first thought.  Correct me if I'm wrong, but it looks like the __new__  function for JavaObject also has a problem.  If you pass through a different port in the Core object, then this function needs to look into the args to see if the are is an instance of Core, and use the  port and timeout values in the core  instead of the default values to create the bridge.  Is this correct?

Carl

**Comment by henrypinkard** [comment]
fixed by #403
